the_well/benchmark/models/unet_classic/README.md

```diff
@@ -0,0 +1,82 @@
+---
+arxiv: 1505.04597
+datasets:
+- polymathi-ai/acoustic_scattering_discontinuous
+- polymathi-ai/acoustic_scattering_inclusions
+- polymathi-ai/acoustic_scattering_maze
+- polymathi-ai/active_matter
+- polymathi-ai/convective_envelope_rsg
+- polymathi-ai/gray_scott_reaction_diffusion
+- polymathi-ai/helmholtz_staircase
+- polymathi-ai/MHD_64
+- polymathi-ai/planetswe
+- polymathi-ai/post_neutron_star_merger
+- polymathi-ai/rayleigh_benard
+- polymathi-ai/rayleigh_taylor_instability
+- polymathi-ai/shear_flow
+- polymathi-ai/supernova_explosion_64
+- polymathi-ai/turbulence_gravity_cooling
+- polymathi-ai/turbulent_radiative_layer_2D
+- polymathi-ai/viscoelastic_instability
```

Comment:
Is there a way we can have a centralized definition for all of these? Otherwise we need to do a [shotgun surgery](https://refactoring.guru/smells/shotgun-surgery) whenever we add a new one

Reply:
Actually the list of the datasets should change with the checkpoint as we have one checkpoint for one training on one dataset. I'll try automate the dataset selection.